### ISSUE TITLE
New rule no-multiple-empty-lines.

### ISSUE THREAD

**Issue by Stanislavsky69** [vuejs/eslint-plugin-vue#1873]
**Please describe what the rule should do:**
The rule should indicate unnecessary line breaks. There should be a setting for a specific block, for example, template. 

**What category should the rule belong to?**

[X] Enforces code style (layout)
[X] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue

/** Error **/
<component></component>

<component></component>

/**Valid **/

<component></component>
<component></component>

/** Error **/

<div>

<component></component>

<component></component>
</div>


/**Valid **/

<div>
<component></component>
<component></component>
</div>


/** Error **/

<div>

    <component></component>

    <component></component>
</div>


/**Valid **/

<div>
    <component></component>
    <component></component>
</div>
```

**Additional context**
https://eslint.org/docs/rules/no-multiple-empty-lines

**Comment by ota-meshi** [comment]
Thank you for posting this issue.
I think the rule you proposed is the same as #1832.
